release-content/0.16/release-notes/17840_Parallel_Transform_Propagation.md

```diff
@@ -1,4 +1,31 @@
-<!-- Parallel Transform Propagation -->
-<!-- https://github.com/bevyengine/bevy/pull/17840 -->
+Transforms in Bevy (and other 3D software) come in two flavors:
 
-<!-- TODO -->
+1. Global transforms ([`GlobalTransform`]), which represent the absolute world-space position, scale and rotation of an object.
+2. Local transforms ([`Transform`]), which represent the position, scale and rotation of an object relative to its parent.
+
+In order to compute the global transform of each object (which is what rendering and physics care about!),
+we need to recursively combine the local transforms of all of our objects down the parent-child hierarchy.
+This process, known as transform propagation, has always been one of the most computationally intensive operations in most Bevy applications.
+As it turns out, most entities in your game are going to have a transform!
+
+This made transform propagation a prime candidate for some serious optimization work, and Bevy 0.16 comes with *two* impressive performance optimizations.
+
+The first optimization improves our parallelization strategies. While we were already splitting the work across threads,
+better work sharing, parallelization across trees and a leaf vs non-leaf split to optimize cache coherency made a huge difference.
```

Comment:
I'd mention this was originally R&D'd downstream in `big_space`, so a good chunk of these optimizations are technically upstreaming. :P